DEVELOPMENT.md

```diff
@@ -0,0 +1,108 @@
+
+## Setting a Route from Notehub to your local instance
```

Comment:
Presumably as that gets merged this step will become required? If so we’ll want to list this as a required setup step in our README.

Also I can make a pass through this doc after it’s merged. Don’t think it’s worth getting into small things like formatting in this PR.